config/routes.rb

```diff
@@ -1,5 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "categories#index"
+  
+  get "/categories", to:"categories#index"
+
+  get "/new_categories", to: "categories#new_categories"
+
+  resources :categories 
```

Comment:
You shouldn't need the other 2 get routes if you have this one. reference: https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default